reference/metadata_tables.rst

```diff
@@ -167,3 +173,65 @@ Citus manages shard health on a per-placement basis and automatically marks a pl
++----------------+----------------------+---------------------------------------------------------------------------+
+| noderack       |        text          | | Rack placement information for the worker node (if available).          |
++----------------+----------------------+---------------------------------------------------------------------------+
+
+::
+
+    SELECT * from pg_dist_node;
+     nodeid | groupid | nodename  | nodeport | noderack | hasmetadata 
+    --------+---------+-----------+----------+----------+-------------
+          1 |       1 | localhost |    12345 | default  | f
+          2 |       2 | localhost |    12346 | default  | f
+          3 |       3 | localhost |    12347 | default  | f
+    (3 rows)
+
+
+Colocation information table
+---------------------------------------
+
+The pg_dist_colocation table contains information about which tables are physically co-located together. When creating tables, the user can specify a table to be co-located physically with another one. Citus then ensures physical co-location of the shards, and can perform optimizations like co-located joins without data shuffling and support features like distributed rollups and enforcing foreign-key constraints. Co-location can also be inferred if the shard-counts, replication factor and partition-column types are the same for two different tables.
+
++------------------------+----------------------+---------------------------------------------------------------------------+
+|      Name              |         Type         |       Description                                                         |
++========================+======================+===========================================================================+
+| colocationid           |         int          | | Unique identifier for a given co-location group.                        |
++------------------------+----------------------+---------------------------------------------------------------------------+
+| shardcount             |         int          | | Shard count for all tables in the given co-location group.              |
++------------------------+----------------------+---------------------------------------------------------------------------+
+| replicationfactor      |         int          | | Replication factor for all tables in the given co-location group.       |
++------------------------+----------------------+---------------------------------------------------------------------------+
+| distributioncolumntype |         oid          | | The type of the distribution column for all tables in the given         |
+|                        |                      | | co-location group                                                       |
++------------------------+----------------------+---------------------------------------------------------------------------+
+
+::
+
+    select * from pg_dist_colocation;
```

Comment:
The rest of this document uses an uppercase `SELECT` (though, weirdly, not uppercase `FROM`). At minimum, change to uppercase `SELECT`.